packages/notifi-react-card/lib/components/subscription/SubscriptionCardV1.tsx

```diff
@@ -191,7 +191,7 @@ export const SubscriptionCardV1: React.FC<SubscriptionCardV1Props> = ({
 
       return { state: 'history' };
     });
-  }, []);
+  }, [demoPreview]);
```

Comment:
This `useEffect` section is supposed to only render once when the card is mounted. It is a potential risk to add the `demoPreview` into dependency to make this view re-rendered.
Do you think we can make the card rerendered in upper level in DAPP (AP) instead of in card level.